dashboard/src/modules/components/SidebarComponent/sideMenuOptions.js

```diff
@@ -42,13 +42,12 @@ export const menuOptions = [
         submenuOf: "tools",
         link: "search",
       },
-      {
-        name: "Pbench Kibana",
-        submenu: true,
-        key: "pbench_kibana",
-        submenuOf: "tools",
-        link: "#",
-      },
     ],
   },
 ];
+
+export const menuOptionsNonLoggedIn = [
+  { key: "dashboard", link: "dashboard", name: "Dashboard" },
+  { key: "search", link: "#", name: "Search" },
+  { key: "expore", link: "#", name: "Explore" },
```

Comment:
I'm uncomfortable with the `"#"` link values.  I assume that these are to-be-implemented features (but nothing indicates that in the code), and I have to wonder whether it wouldn't be better to just omit them (or leave them in the code but commented-out).

The problem with including them is that the user will see them, click on them, and then wonder why nothing happens (right??).  If we really want them in the code, then they should provide the user with feedback which says, "not yet implemented" or "coming soon" or something.

Also, should the `"Search"` menu item at line 51 really be `"#"`?  Shouldn't it do the same thing as the menu item at line 43?

Reply:
When the user clicks on these items, it will be redirected to "Coming soon" page